components/layout/ModPage.vue

```diff
@@ -531,7 +533,18 @@ export default {
 
 .mod-navigation {
   @extend %card-spaced-b;
-  padding-bottom: 0.2rem;
+
+  .tabs {
+    overflow-x: auto;
+  }
+
+  .settings-tab {
+    margin-left: auto;
+    @media screen and (max-width: 1024px) {
+      margin-left: 0.5rem;
+      padding-right: 2rem;
```

Comment:
This is very weird on mobile
![image](https://i.imgur.com/mm6Gk36.png)

Reply:
Everything should be looking better with 8411e0d40bdd77abfd7768dceb72c1874f459344.

Old:
![image](https://user-images.githubusercontent.com/44736536/117619483-2a353080-b124-11eb-9046-8f820db8d9e0.png)

Now:
![image](https://user-images.githubusercontent.com/44736536/117619358-040f9080-b124-11eb-96e5-8c79a360c6ac.png)

Mobile:
![image](https://user-images.githubusercontent.com/44736536/117619913-b5aec180-b124-11eb-99a3-afe21321709c.png)